pkg/apis/kudo/v1beta1/instance_types_helpers.go

```diff
@@ -186,18 +181,18 @@ func (i *Instance) GetOperatorVersion(c client.Reader) (ov *OperatorVersion, err
 	return GetOperatorVersionByName(i.Spec.OperatorVersion.Name, i.OperatorVersionNamespace(), c)
 }
 
-func GetOperatorVersionByName(ovn, ns string, c client.Reader) (ov *OperatorVersion, err error) {
-	ov = &OperatorVersion{}
-	err = c.Get(context.TODO(),
-		types.NamespacedName{
-			Name:      ovn,
-			Namespace: ns,
-		},
-		ov)
-	if err != nil {
-		return nil, err
+// IsChildInstance method return true if this instance is owned by another instance (as a dependency) and false otherwise
+func (i *Instance) IsChildInstance() bool {
+	for _, or := range i.GetOwnerReferences() {
+		if or.Kind == i.Kind {
```

Comment:
How does having the same kind determine ownership here? Can you add a comment explaining this? Why not compare the `UID` fields?

Reply:
Oh, it's about that this is owned by _any_ other instance, not a specific one. That's already explained well enough in the function description. Resolving this.

Reply:
Basically, if there is any owner with the same kind `Instance` then this Instance is owned by another one.